symfony/plugins/orangehrmAdminPlugin/config/routes.yaml

```diff
@@ -433,3 +456,18 @@ apiv2_admin_pay_grade:
   methods: [ GET, PUT ]
   defaults:
     _api: OrangeHRM\Admin\Api\PayGradeAPI
+
+view_work_shift:
+  path: /admin/viewWorkShifts
+  controller: OrangeHRM\Admin\Controller\WorkShiftController::handle
+  methods: [ GET ]
+
+save_work_shift:
+  path: /admin/saveWorkShifts
```

Comment:
we may have to block screen for non admin users since this URL were not there in 4.x